Framework/Kernel/test/GlobTest.h

```diff
@@ -11,8 +11,9 @@
 #include "MantidKernel/ConfigService.h"
 #include "MantidKernel/Glob.h"
 
-#include <Poco/File.h>
 #include <Poco/Path.h>
```

Comment:
Header file is not required.

Reply:
This is a bigger change to remove. This PR is meant to be a positive step rather than taking care of everything.